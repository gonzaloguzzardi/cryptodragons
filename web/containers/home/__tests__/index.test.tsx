import renderer from 'react-test-renderer'
import IndexPage from '../'

describe('Index page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<IndexPage deviceType="desktop" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
