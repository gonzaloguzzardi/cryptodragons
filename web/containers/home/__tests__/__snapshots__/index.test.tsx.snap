// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<div
  className="container"
>
  <main>
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <a
          className="homeIconLabelAnchor"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="MuiAvatar-root MuiAvatar-circle"
          >
            <img
              alt="CryptoDragons Icon"
              className="MuiAvatar-img"
              src="/assets/dragonsito.jpg"
            />
          </div>
        </a>
        <a
          className="appbarActionLink"
          href="/my-dragons"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <button
            aria-label="my-dragons"
            className="MuiButtonBase-root MuiIconButton-root appbarActionLink_Button"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root appbarActionLink_Icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="4.5"
                  cy="9.5"
                  r="2.5"
                />
                <circle
                  cx="9"
                  cy="5.5"
                  r="2.5"
                />
                <circle
                  cx="15"
                  cy="5.5"
                  r="2.5"
                />
                <circle
                  cx="19.5"
                  cy="9.5"
                  r="2.5"
                />
                <path
                  d="M17.34 14.86c-.87-1.02-1.6-1.89-2.48-2.91-.46-.54-1.05-1.08-1.75-1.32-.11-.04-.22-.07-.33-.09-.25-.04-.52-.04-.78-.04s-.53 0-.79.05c-.11.02-.22.05-.33.09-.7.24-1.28.78-1.75 1.32-.87 1.02-1.6 1.89-2.48 2.91-1.31 1.31-2.92 2.76-2.62 4.79.29 1.02 1.02 2.03 2.33 2.32.73.15 3.06-.44 5.54-.44h.18c2.48 0 4.81.58 5.54.44 1.31-.29 2.04-1.31 2.33-2.32.31-2.04-1.3-3.49-2.61-4.8z"
                />
              </svg>
            </span>
          </button>
        </a>
        <a
          className="appbarActionLink"
          href="/marketplace"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <button
            aria-label="marketplace"
            className="MuiButtonBase-root MuiIconButton-root appbarActionLink_Button"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root appbarActionLink_Icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M21.9 8.89l-1.05-4.37c-.22-.9-1-1.52-1.91-1.52H5.05c-.9 0-1.69.63-1.9 1.52L2.1 8.89c-.24 1.02-.02 2.06.62 2.88.08.11.19.19.28.29V19c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-6.94c.09-.09.2-.18.28-.28.64-.82.87-1.87.62-2.89zm-2.99-3.9l1.05 4.37c.1.42.01.84-.25 1.17-.14.18-.44.47-.94.47-.61 0-1.14-.49-1.21-1.14L16.98 5l1.93-.01zM13 5h1.96l.54 4.52c.05.39-.07.78-.33 1.07-.22.26-.54.41-.95.41-.67 0-1.22-.59-1.22-1.31V5zM8.49 9.52L9.04 5H11v4.69c0 .72-.55 1.31-1.29 1.31-.34 0-.65-.15-.89-.41-.25-.29-.37-.68-.33-1.07zm-4.45-.16L5.05 5h1.97l-.58 4.86c-.08.65-.6 1.14-1.21 1.14-.49 0-.8-.29-.93-.47-.27-.32-.36-.75-.26-1.17zM5 19v-6.03c.08.01.15.03.23.03.87 0 1.66-.36 2.24-.95.6.6 1.4.95 2.31.95.87 0 1.65-.36 2.23-.93.59.57 1.39.93 2.29.93.84 0 1.64-.35 2.24-.95.58.59 1.37.95 2.24.95.08 0 .15-.02.23-.03V19H5z"
                />
              </svg>
            </span>
          </button>
        </a>
        <a
          className="appbarActionLink"
          href="/guides"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <button
            aria-label="guides"
            className="MuiButtonBase-root MuiIconButton-root appbarActionLink_Button"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root appbarActionLink_Icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M21 5c-1.11-.35-2.33-.5-3.5-.5-1.95 0-4.05.4-5.5 1.5-1.45-1.1-3.55-1.5-5.5-1.5S2.45 4.9 1 6v14.65c0 .25.25.5.5.5.1 0 .15-.05.25-.05C3.1 20.45 5.05 20 6.5 20c1.95 0 4.05.4 5.5 1.5 1.35-.85 3.8-1.5 5.5-1.5 1.65 0 3.35.3 4.75 1.05.1.05.15.05.25.05.25 0 .5-.25.5-.5V6c-.6-.45-1.25-.75-2-1zm0 13.5c-1.1-.35-2.3-.5-3.5-.5-1.7 0-4.15.65-5.5 1.5V8c1.35-.85 3.8-1.5 5.5-1.5 1.2 0 2.4.15 3.5.5v11.5z"
                />
                <path
                  d="M17.5 10.5c.88 0 1.73.09 2.5.26V9.24c-.79-.15-1.64-.24-2.5-.24-1.7 0-3.24.29-4.5.83v1.66c1.13-.64 2.7-.99 4.5-.99zM13 12.49v1.66c1.13-.64 2.7-.99 4.5-.99.88 0 1.73.09 2.5.26V11.9c-.79-.15-1.64-.24-2.5-.24-1.7 0-3.24.3-4.5.83zM17.5 14.33c-1.7 0-3.24.29-4.5.83v1.66c1.13-.64 2.7-.99 4.5-.99.88 0 1.73.09 2.5.26v-1.52c-.79-.16-1.64-.24-2.5-.24z"
                />
              </svg>
            </span>
          </button>
        </a>
        <div
          className="profileSection"
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              Start
            </span>
          </button>
        </div>
      </div>
    </header>
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      id="back-to-top-anchor"
    />
    <div
      className="makeStyles-root-1"
      onClick={[Function]}
      role="presentation"
      style={
        Object {
          "transform": "scale(0)",
          "visibility": "hidden",
        }
      }
    >
      <button
        aria-label="Scroll back to top"
        className="MuiButtonBase-root MuiFab-root MuiFab-secondary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiFab-label"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        className="MuiBox-root MuiBox-root-2"
      >
        Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
              Mobile Mobile Mobile Mobile Mobile Mobile Mobile Mobile
      </div>
    </div>
  </main>
</div>
`;
